config/environment.js

```diff
@@ -20,6 +20,10 @@ module.exports = function(environment) {
 
     segmentioKey: 'qxwhj8ys4t',
 
+    externalUrls: {
+      gettingStartedDocs: 'https://support.aptible.com/hc/en-us/articles/202638630-Deploying-your-first-app'
+    },
```

Comment:
If it is expected that we will have a number of these, we should consider just putting them in a `utils` file. We have a configurable list of `aptibleHosts` already, and the rest of the path seems unlikely to change from env to env.

Data in the `config/environment.js` is sent to the client at least twice, as we've discovered. Keeping it slim should be on our minds.